shared/types.ts

```diff
@@ -5,4 +5,8 @@ export type TStudent = {
 	email: string;
 	updated_at: string;
 	created_at: string;
-};
\ No newline at end of file
+};
+
+export type TSelectedLanguage = {
+	languageCode: string;
```

Comment:
mettre uniquement les 2 cas au lieu de string : 
```suggestion
	languageCode: "en-EN" | "fr-FR";
```